crawl-ref/source/spl-goditem.cc

```diff
@@ -1190,6 +1190,7 @@ void torment_player(actor *attacker, torment_source_type taux)
     {
     case TORMENT_CARDS:
     case TORMENT_SPELL:
+    case TORMENT_CARDS_PAIN:
```

Comment:
```suggestion
    case TORMENT_CARD_PAIN:
```

`TORMENT_CARDS` shouldn't have been plural; but we don't have to repeat past mistakes.